### DISCUSSION TITLE
Analyticsの表示がおかしい

### DISCUSSION THREAD

**Discussion by riya-amemiya** [lapras-inc/lapras-user-community#14] [General]
[Portfolio](https://lapras.com/person)のAnalyticsを押すと同じ企業が複数表示されるバグがある。
このバグが発生してる方他にいますか？

例

- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社
- 2023.03.20 hoge株式会社

**Comment by UserSuccessTeamLAPRAS** [comment]
LAPRAS運営です。
いつもLAPRASをご利用いただきありがとうございます！😄
PortfolioのAnalytics機能につきましては、同日に複数回アクセスがあった場合に複数ログが表示されるようになっております。
アクセス数が多過ぎる場合は何らかの問題が発生している場合がございますので、念の為調査を行いますね！

**Comment by UserSuccessTeamLAPRAS** [comment]
@riya-amemiya 

LAPRAS運営です！
先日は仕様についてのご指摘ありがとうございました 😄 
社内で話したところ、不具合に見える直感的でない表示になっているのは良くない！ということで、同じ会社の訪問は1件にまとまるように改修いたしました。

内容はこちらのNewsをご覧ください。
https://news.intercom.com/lapras/news/16114

この度はLAPRASへのご指摘ありがとうございます。今後もお気づきの点があれば、コミュニティ等の窓口からお気軽にご意見ご感想をいただけると嬉しいです！ 🙏
